packages/bits/demo/src/components/demo/wizard-v2/wizard-step-change-example/wizard-step-change-example.component.ts

```diff
@@ -0,0 +1,43 @@
+import {
+    AfterViewInit,
+    Component,
+    TemplateRef,
+    ViewChild,
+} from "@angular/core";
+
+import { IWizardStepData } from "../wizard-dynamic/wizard-dynamic.example.component";
+import {StepperSelectionEvent} from "@angular/cdk/stepper";
+import {ToastService} from "@nova-ui/bits";
+
+@Component({
+    selector: "nui-wizard-step-change-example",
+    templateUrl: "./wizard-step-change-example.component.html",
+})
+export class WizardStepChangeExampleComponent implements AfterViewInit {
+    public steps: IWizardStepData[] = [];
```

Comment:
Hi, Dima. I didn't notice before that we're importing an interface from another example. Do you mind duplicating the interface in the example instead of importing it? That way the example is self-contained instead of having a mystery interface.

Reply:
Hi Blake! 
I'll fix it now

Reply:
@Blakesalot fixed